### ISSUE TITLE
Deployment Failing

### ISSUE THREAD

**Issue by jamesgilmorelyst** [farridav/django-jazzmin#541]
We recently deployed v2.6.1 of Django-Jazzmin, but we failed to publish the wheels to PyPi:
https://github.com/farridav/django-jazzmin/actions/runs/8381641996/job/22953651571

The error we got (despite it being masked as a success) was:
```
HTTP Error 403: Username/Password authentication is no longer supported. Migrate to API Tokens or Trusted Publishers instead. See https://test.pypi.org/help/#apitoken and https://test.pypi.org/help/#trusted-publishers | b'<html>\n <head>\n  <title>403 Username/Password authentication is no longer supported. Migrate to API Tokens or Trusted Publishers instead. See https://test.pypi.org/help/#apitoken and https://test.pypi.org/help/#trusted-publishers\n \n <body>\n  <h1>403 Username/Password authentication is no longer supported. Migrate to API Tokens or Trusted Publishers instead. See https://test.pypi.org/help/#apitoken and https://test.pypi.org/help/#trusted-publishers\n  Access was denied to this resource.<br/><br/>\n\n\n\n \n'
```

We need to update the account in Test PyPi and Prod PyPi to use API tokens instead, update our secrets in this repo to add the API token, and update the GHA to use the token instead of the deprecated username and password fields.

Sadly I don't have permissions to any of these platforms to do the work myself.

cc: @farridav @TysonRV

**Comment by farridav** [comment]
I will try to get into this over the next few days

**Comment by jamesgilmorelyst** [comment]
Thanks very much 🙂

**Comment by jamesgilmorelyst** [comment]
Fixed now and 2.6.1 has been released to PyPi
